README.md

```diff
@@ -1 +1 @@
-test
\ No newline at end of file
+(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧
```

Comment:
This is cute and all, but please look up Github Markdown language to make this look more official. Tons of templates online. Describe your drawing app

Reply:
Updated with new text!